x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/ilm_phase_filter/styles.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFormControlLayout } from '@elastic/eui';
+import styled from 'styled-components';
+
+export const StyledOption = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const StyledOptionLabel = styled.span`
+  font-weight: ${({ theme }) => theme.eui.euiFontWeightBold};
+`;
+
+export const StyledFormControlLayout = styled(EuiFormControlLayout)`
+  height: 42px;
```

Comment:
Here I think the `theme.eui.euiSize...` could be used

Reply:
Tbh I don't have the context of this change, since I didn't do it, this came over from the old code. I assume this was an attempt at fixing the height of EuiComboBox, so that it doesn't swallow the border below. So I would abstain from making it dynamic, because it might not match the measure once changed. Wdyt?